test/specs/ExpectAttributeValueSpec.ts

```diff
@@ -0,0 +1,37 @@
+import { expect } from 'chai';
+import { BrowserUtils } from '../../src/commons/BrowserUtils';
+import { describeCommon } from '../TestHelper';
+
+const SELECTOR_WITH_ATTRIBUTE: string =
+  '//button[@id="button_with_attribute_value"]';
+const ATTRIBUTE_NAME: string = 'value';
+const CORRECT_ATTRIBUTE_VALUE: string = 'hello world';
+const INCORRECT_ATTRIBUTE_VALUE: string = 'hello hello world';
+
+/**
+ * wdio-allure-ts expectAttributeValueSpec
+ */
+describeCommon('expectAttributeValue', () => {
```

Comment:
add more test cases
- incorrect/not existing attribute name
- incorrect selector

Reply:
done